### PR TITLE
Fix: Make use of __DIR__ constant

### PR DESCRIPTION
**Pull request by localheinz**
This PR

* [x] makes use of the `__DIR__` constant when referencing file paths

### CODE REVIEW

File: src/public/index.php [hunk 1]
```diff
@@ -9,7 +9,7 @@
 
 include __DIR__ . '/../../vendor/autoload.php';
 if (!function_exists('apache_request_headers')) {
-    include '../inc/nginx-helper.php';
+    include __DIR__ . '/../inc/nginx-helper.php';
```
Comment (localheinz) [inline src/public/index.php]: This should probably be `require`.

File: src/public/index.php [hunk 2]
```diff
@@ -41,7 +41,7 @@ function handle_exception(\Throwable $e)
 // config setup
 define('BASEPATH', '.');
 $config = [];
-include '../config.php';
+include __DIR__. '/../config.php';
```
Comment (localheinz) [inline src/public/index.php]: This should probably be `require`.

File: src/public/index.php [hunk 3]
```diff
@@ -51,7 +51,7 @@ function handle_exception(\Throwable $e)
 
 // database setup
 $db = [];
-include '../database.php';
+include __DIR__ . '/../database.php';
```
Comment (localheinz) [inline src/public/index.php]: This should probably be `require`.

**Comment by svpernova09** [comment]
Thanks for these changes.

**Comment by localheinz** [comment]
Thank you, @svpernova09!

**Pull request merged**